src/block/block_ckpt.c

```diff
@@ -389,14 +389,18 @@ static int
 __ckpt_process(WT_SESSION_IMPL *session, WT_BLOCK *block, WT_CKPT *ckptbase)
 {
 	WT_BLOCK_CKPT *a, *b, *ci;
+	WT_CONNECTION_IMPL *conn;
 	WT_CKPT *ckpt, *next_ckpt;
 	WT_DECL_ITEM(tmp);
 	WT_DECL_RET;
 	uint64_t ckpt_size;
+	uint64_t time_diff;
```

Comment:
This declaration can be along with the previous one.

Reply:
The two are not associated , hence kept seperate.

Reply:
Our coding standard puts them together, being the same type, on the declaration line anyway.